src/ConcreteOperations/issubset.jl

```diff
@@ -152,15 +152,20 @@ function ⊆(H1::AbstractHyperrectangle{N},
            witness::Bool=false
           )::Union{Bool, Tuple{Bool, Vector{N}}} where {N<:Real}
     @assert dim(H1) == dim(H2)
-
-    for i in 1:dim(H1)
-        c_dist = center(H1)[i] - center(H2)[i]
+    zero_N = zero(N)
+    c1 = center(H1)
+    c2 = center(H2)
+    @inbounds for i in 1:dim(H1)
+        c_dist = c1[i] - c2[i]
         r_dist = radius_hyperrectangle(H1, i) - radius_hyperrectangle(H2, i)
-        if -r_dist < c_dist || c_dist < r_dist
+        # check if c_dist is in the interval [r_dist, -r_dist]
+        left = _leq(r_dist, c_dist)
+        right = _leq(c_dist, -r_dist)
+        if !left || !right
```

Comment:
This is a (minor) performance degradation. Previously the condition was short-circuited if the first disjunct was satisfied. Now you evaluate `right` in all cases.
I would prefer to write comments that explain every condition (in particular because `left`/`right` are very ambiguous and do not help me) or write helper functions like `_in_left_bound`.

Reply:
Ok i will write the statements in the same line, good point. (`left` and `right` mean left side of the inequality and right side of the inequality respectively).